internal/comm/httptransport.go

```diff
@@ -133,13 +136,18 @@ func (p *HTTPTransport) Start() error {
 		return errors.Wrapf(err, "failed to start rafthttp transport")
 	}
 
+	var membersList []*types.PeerConfig
 	for _, peer := range p.clusterConfig.ConsensusConfig.Members {
 		if peer.RaftId != p.raftID {
+			membersList = append(membersList, peer)
 			p.transport.AddPeer(
 				etcd_types.ID(peer.RaftId),
 				[]string{fmt.Sprintf("http://%s:%d", peer.PeerHost, peer.PeerPort)}) //TODO unsecure for now, add TLS/https later
 		}
 	}
+	if err = p.catchUpClient.UpdateMembers(membersList); err != nil {
```

Comment:
Sure, but dynamic updates are not supported yet, just a static cluster. This will be dealt with in 
https://github.com/ibm-blockchain/bcdb-server/issues/46